Project settings metadata template creation is broken

## Describe the bug
Server error occurs when trying to create a new metadata template from the project settings metadata template page.

## Steps to reproduce the problem
1. Went to project/settings/"Sample Metadata Templates" page
2. Clicked "Create New Template" and added a couple fields
3. Clicked Save
4. Got a "Server Error" page

## Expected behaviour
The template to be created and redirected back to settings page

## Additional context
Exception thrown on server:
```
05 Apr 2019 10:08:42,722 ERROR ca.corefacility.bioinformatics.irida.ria.web.errors.ExceptionHandlerController:117 - org.hibernate.TransientObjectException: object references an unsaved transient instance - save the transient instance before flushing: ca.corefacility.bioinformatics.irida.model.sample.MetadataTemplateField; nested exception is java.lang.IllegalStateException: org.hibernate.TransientObjectException: object references an unsaved transient instance - save the transient instance before flushing: ca.corefacility.bioinformatics.irida.model.sample.MetadataTemplateField
org.springframework.dao.InvalidDataAccessApiUsageException: org.hibernate.TransientObjectException: object references an unsaved transient instance - save the transient instance before flushing: ca.corefacility.bioinformatics.irida.model.sample.MetadataTemplateField; nested exception is java.lang.IllegalStateException: org.hibernate.TransientObjectException: object references an unsaved transient instance - save the transient instance before flushing: ca.corefacility.bioinformatics.irida.model.sample.MetadataTemplateField
...
at ca.corefacility.bioinformatics.irida.service.impl.sample.MetadataTemplateServiceImpl$$EnhancerBySpringCGLIB$$ecf6e586.createMetadataTemplateInProject(<generated>)
	at ca.corefacility.bioinformatics.irida.ria.web.projects.metadata.ProjectSamplesMetadataTemplateController.saveMetadataTemplate(ProjectSamplesMetadataTemplateController.java:117)
...
Caused by: java.lang.IllegalStateException: org.hibernate.TransientObjectException: object references an unsaved transient instance - save the transient instance before flushing: ca.corefacility.bioinformatics.irida.model.sample.MetadataTemplateField
...
Caused by: org.hibernate.TransientObjectException: object references an unsaved transient instance - save the transient instance before flushing: ca.corefacility.bioinformatics.irida.model.sample.MetadataTemplateField
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,6 +1,10 @@
 Changes
 =======
 
+19.05 to 19.09
+--------------
+* [UI]: Fixed bug where a new metadata template could not be created. (19.05.1)
+
 19.01 to 19.05
 ---------------
 * [Documentation]: Added the CalVer updates to the documentation getting started guide.

--- a/pom.xml
+++ b/pom.xml
@@ -4,7 +4,7 @@
 	<groupId>ca.corefacility.bioinformatics</groupId>
 	<artifactId>irida</artifactId>
 	<packaging>war</packaging>
-	<version>19.05</version>
+	<version>19.05.1</version>
 	<name>irida</name>
 	<url>http://www.irida.ca</url>
 

--- a/src/main/java/ca/corefacility/bioinformatics/irida/ria/web/projects/metadata/ProjectSamplesMetadataTemplateController.java
+++ b/src/main/java/ca/corefacility/bioinformatics/irida/ria/web/projects/metadata/ProjectSamplesMetadataTemplateController.java
@@ -101,7 +101,9 @@ public class ProjectSamplesMetadataTemplateController {
 		for (String field : fields) {
 			MetadataTemplateField f = metadataTemplateService.readMetadataFieldByLabel(field);
 			if (f == null) {
-				templateFields.add(new MetadataTemplateField(field, "text"));
+				MetadataTemplateField templateField = metadataTemplateService.saveMetadataField(
+						new MetadataTemplateField(field, "text"));
+				templateFields.add(templateField);
 			} else {
 				templateFields.add(f);
 			}

